tools/booster/Proxy.hs

```diff
@@ -219,6 +219,61 @@ respondEither mbStatsVar booster kore req = case req of
             -- can only be an error at this point
             res -> pure res
 
+    simplifyExecuteState :: Maybe Text -> ExecuteState -> m ExecuteState
+    simplifyExecuteState mbModule s = do
+        Log.logInfoNS "proxy" "Simplifying execution state"
+        simplResult <-
+            handleSimplify $ toSimplifyRequest s
```

Comment:
If we use `handleSimplify` here, the statistics will include all those simplification calls as if they were regular requests. However, this may be the best alternative, otherwise we would need to reflect this `simplify-and-continue` in the statistics code.